packages/runtime/lib/worker/app.js

```diff
@@ -87,7 +86,7 @@ class PlatformaticApp extends EventEmitter {
     if (config.plugins !== undefined && this.#watch && watch.enabled !== false) {
       /* c8 ignore next 4 */
       this.#debouncedRestart = debounce(() => {
-        this.server.log.info('files changed')
+        this.stackable.log('files changed', { level: 'debug' })
```

Comment:
Why this change?

Reply:
because we don't have access to the platformatic app (fastify app) anymore. If we want to ask stackable to log something we should do it through the interface

Reply:
I see, makes totally sense. In that case I would opt for a single object then, like:

```
this.stackable.log({message: 'files changed', level: 'debug'})
```

Reply:
Also, don't we need the log function in `StackableInterface`?

Reply:
good point, added

Reply:
What about using a single object with `message` inside? So we don't assume people always will send a string.

Reply:
What people? This method is needed so that runtime can log something in a service thread.

Reply:
On this side yes, but on the other side, people that write stackables might not always be using strings in their logs.